gui/wxpython/modules/import_export.py

```diff
@@ -241,6 +243,36 @@ def _getBlackListedFlags(self):
         """Get flags which will not be showed in Settings page"""
         raise NotImplementedError()
 
+    def _nameValidationFailed(self, layers_list):
+        """Output map name validation callback
+
+        :param layers_list: LayersList class instance
+        """
+        if isinstance(layers_list.output_map, list):
+            maps = [
+                '<{}>'.format(m) for m in layers_list.output_map]
+            message = _(
+                "Output maps names %(names)s exist. "
```

Comment:
It should be "Output map names" I think.

Reply:
I fixed it.